DAC_configuration.md

```diff
@@ -0,0 +1,21 @@
+# DAC Configuration
+
+If you are using a DAC board, you will need to make a couple of changes to the device configuration in the balenaCloud dashboard.
+
+* Disable the on-board audio by editing the existing RESIN_HOST_CONFIG_dtparam variable to set ”audio=off”.
+* Add an additional custom configuration variable called BALENA_HOST_CONFIG_dtoverlay. The value of this will depend on your DAC board.
```

Comment:
```suggestion
* Add an additional custom configuration variable called `BALENA_HOST_CONFIG_dtoverlay`. The value of this will depend on your DAC board.
```